eng/vmr-sync.sh

```diff
@@ -89,6 +89,8 @@ vmr_dir=''
 vmr_branch='main'
 target_ref=''
 verbosity=verbose
+tpn_template=''
```

Comment:
Please update the documentation in the header of this file. The documentation actually gets printed if you pass `-h`

Reply:
I should also make them optional

Reply:
Let's not, I think we should make sure we're not forgetting to update the readme's anywhere